website/blog/2023-06-04-databend-weekly-96.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "This Week in Databend #96"
+date: 2023-06-04
+slug: 2023-06-04-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: b41sh
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: Dousir9
+  - name: everpcpc
+  - name: gitccl
+  - name: JackTan25
+  - name: jun0315
+  - name: leiysky
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: tisonkun
+  - name: Xuanwo
+  - name: xudong963
+  - name: zhang2014
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Virtual Column
+
+One way to optimize querying JSON internal fields is to create virtual columns for these fields using `CREATE VIRTUAL COLCUM`. After creating the virtual columns, we can generate them using `GENERATE VIRTUAL COLUMN` function. Once these steps are completed, we can directly access data in the JSON internal fields through their corresponding virtual columns.
+
+Databend now supports a series of syntax for creating, dropping, generating and altering virtual columns. Here is a simple example:
+
+```sql
+create table test (id int, val json);
+insert into test values(1, '{"a":33,"b":44}'),(2, '{"a":55,"b":66}');
+create virtual columns (val['a'], val['b']) for test;
+generate virtual columns for test;
+
+select val['a'], val['b'] from test;
++----------+----------+
+| val['a'] | val['b'] |
++----------+----------+
+| 33       | 44       |
+| 55       | 66       |
++----------+----------+
+```
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [RFC | JSON Optimization - Virtual Column](https://databend.rs/doc/contributing/rfcs/json-optimization#virtual-column)
+- [PR | feat(query): Support generate virtual columns](https://github.com/datafuselabs/databend/pull/11590)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Interactive Documents Powered by Askbend
+
+Do you remember [AskBend](https://github.com/datafuselabs/askbend)? It is a Knowledge Base Search and Completion project created and maintained by the Databend team, combining the simplicity of Databend with the magic of OpenAI.
```

Comment:
combining the simplicity of Databend with the magic of OpenAI.   --------Im not sure I understand "simplicity" here